js/effects.js

```diff
@@ -0,0 +1,129 @@
+const scale = document.querySelector('.scale__control--value');
+const imagePreview =  document.querySelector('.img-upload__preview > img');
+const effectsRadio = document.querySelectorAll('.effects__radio');
+const effectLevelValue = document.querySelector('.effect-level__value');
+//const slider = document.querySelector('.effect-level__slider');
+
+
+const buttonUp = document.querySelector('.scale__control--bigger');
+const buttonDown = document.querySelector('.scale__control--smaller');
+
+
+const labels = document.getElementsByTagName('label');
+for (let i = 0; i < labels.length; i++) {
+  if (labels[i].htmlFor !== '') {
+    const elem = document.getElementById(labels[i].htmlFor);
+    if (elem) {
+      elem.label = labels[i];
+    }
+  }
+}
+
+effectLevelValue.addEventListener('input', () => {
+  const style = getComputedStyle(imagePreview);
+  const filter = style.filter.startsWith;
+  switch (filter) {
+    case 'invert': {
+      effectLevelValue.max = 100;
+      effectLevelValue.min = 0;
+      effectLevelValue.step = 1;
+      imagePreview.style.filter = `invert(${effectLevelValue.value}%)`;
+      break;
+    }
+
+    case 'sepia': {
+      effectLevelValue.max = 1;
+      effectLevelValue.min = 0;
+      effectLevelValue.step = 0.1;
+      imagePreview.style.filter = `sepia(${effectLevelValue.value})`;
+      break;
+    }
+
+    case 'blur': {
+      effectLevelValue.max = 3;
+      effectLevelValue.min = 0;
+      effectLevelValue.step = 0.1;
+      imagePreview.style.filter = `blur(${effectLevelValue.value}px)`;
+      break;
+    }
+
+    case 'brightness': {
+      effectLevelValue.max = 3;
+      effectLevelValue.min = 1;
+      effectLevelValue.step = 0.1;
+      imagePreview.style.filter = `brightness(${effectLevelValue.value})`;
+      break;
+    }
+
+    case 'grayscale': {
+      effectLevelValue.max = 1;
+      effectLevelValue.min = 0;
+      effectLevelValue.step = 0.1;
+      imagePreview.style.filter = `grayscale(${effectLevelValue.value})`;
+      break;
+    }
+  }
+});
+
+const addEffect = (input) => {
+  imagePreview.classList.forEach((val, key, list) => list.remove(val));
+  const span = input.label.querySelector('.effects__preview');
```

Comment:
const value = input.getAttribute("value");
const label = document.querySelector('label[for=`${value}`]');